docs/guides/tools/cis-x.md

```diff
@@ -39,6 +39,9 @@ cis-X currently only works with hg19 (GRCh37).
 | [Somatic SNV/indels]         | File      | Tab-delimited file containing somatic SNV/indels in the tumor genome                            | *.txt          |
 | [Somatic SVs]                | File      | Tab-delimited file containing somatic acquired structural variants in the tumor genome          | *.txt          |
 | [Somatic CNVs]               | File      | Tab-delimited file containing copy number aberrations in the tumor genome                       | *.txt          |
+| CNV/LOH action               | String    | The behavior when handling markers in CNV/LOH regions. Can be either `keep` or `drop`.          | drop           |
+| Minimum coverage for WGS     | Integer   | The minimum coverage in WGS to be included in the analysis                                      | 10             |
```

Comment:
is 10 a practical value for this input? (I have no idea) Let's give an example that is a reasonable order of magnitude.

Reply:
Yes, this value was given by the main author. It's also the default in the app.